andela/travel_tool_front
# #162369171 Add e2e test for guesthouse creation.

#### What does this PR do?
End-to-End testing for guesthouse creation.

#### Description of Task to be completed?
* Add test for input validation.
* Add test for guesthouse image upload
* Add test for successful guesthouse creation.
* Add test for validation of guesthouse name against duplicates.

#### How should this be manually tested?
* Clone the repo ```git clone https://github.com/andela/travel_tool_front.git```
* Change directory with ```cd travel_tool_front```
* Checkout to the branch ```ch-create-guest-house-e2e-162369171```
* Run ```yarn start``` to start up the front-end server and ```yarn start:dev``` for the back-end.
* You should have another terminal open with the branch ```ch-create-guest-house-e2e-162369171``` on ```travel_tool_front``` , that is the front-end repo.
* Run the cypress test with `yarn end2end`
* A new window will open with the test files
* Click on `create-guesthouse.spec.js` under the Accommodation folder.
* The e2e tests should run successfully. 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[162369171](https://www.pivotaltracker.com/story/show/162369171)

#### Screenshots (if appropriate)


#### Questions:
